.evergreen/constants.js

```diff
@@ -37,8 +37,8 @@ exports.MONGODB_VERSIONS = [
   { shortName: '70xe', versionSpec: '7.0.x-enterprise' },
   { shortName: '80xc', versionSpec: '8.0.x' },
   { shortName: '80xe', versionSpec: '8.0.x-enterprise' },
-  { shortName: '82rc', versionSpec: '8.2.0-rc4' },
-  { shortName: '82rce', versionSpec: '8.2.0-rc4-enterprise' },
+  { shortName: '82rc', versionSpec: '8.2.0' },
```

Comment:
I guess we should remove the `rc` suffix I added too from shortname 
(in hindsight, I guess this choice ends up messing with task history which in some way makes sense i.e. rc vs non-rc but for our purposes should probably have followed the typical naming scheme from getgo)

Reply:
Yup, done!